docs/experiments-ensemble_kd.md

```diff
@@ -0,0 +1,35 @@
+# Pyserini: Replicating Ensemble_KD MSMARCO Results
```

Comment:
Let's retain the original model name `distilbert-dot-margin_mse-T2` to be consistent, and call it

And use the more human readable name "DistilBERT KD" in the appropriate context, e.g., 

"Pyserini: Replicating DistilBERT KD Results" <--- leave out MS MARCO to be consistent with TCT-ColBERT

Also add link from README (for ANCE also)

Reply:
updated, I'll re-upload the index with name changed later